parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -303,6 +303,17 @@ try
 
         Write-KubeClusterConfig -MasterIP $MasterIP -KubeDnsServiceIp $KubeDnsServiceIp
 
+        Write-Log "Download kubelet binaries and unzip"
+        Get-KubePackage -KubeBinariesSASURL $global:KubeBinariesPackageSASURL
+
+        # The custom package has a few files that are necessary for future steps (nssm.exe)
+        # this is a temporary work around to get the binaries until we depreciate
+        # custom package and nssm.exe as defined in aks-engine#3851.
+        if ($global:WindowsKubeBinariesURL){

```

Comment:
Since you removed the comment `# this overwrite the binaries that are download from the custom packge with binaries`, I want to ask one question "Does AKS need $global:WindowsKubeBinariesURL?"

Reply:
AKS specifies the binary url by ContainerService.Properties.OrchestratorProfile.KubernetesConfig.WindowsNodeBinariesURL

Reply:
I know that aks-engine uses it. Does AKS use it?
You do not need to remove it in this PR but please still keep the comments. I think that we can remove it later.

Reply:
What's the usage here?

Reply:
Will close this comment cause AKS also specify WindowsKubeBinariesURL.